util/configuration.py

```diff
@@ -279,3 +281,33 @@ def update(cls):
                         line = 'rcon.password={}\n'.format(cls.password)
                     
                     fileOut.write(line)
+
+class Temp:
+    __data = File.data.get("Temp", {})
+    current = __data.get('current', None)
+    date = __data.get('date', None)
+    elapsed = __data.get('elapsed', None)
+    maximum = __data.get('maximum', None)
+    minutes = __data.get('minutes', None)
+    overheating = __data.get('overheating', False)
```

Comment:
Can't we convert this into a method? Is it strictly necessary to store this property in our config file?

Reply:
Well, I guess I don't understand the order of operations, which is why I ask. Because I would assume once you've set your current temp, you could have a method to check to see if it is above the maximum temp. Would you mind explaining to me the order of operations?